docs/api/ShallowWrapper/forEach.md

```diff
@@ -30,7 +30,7 @@ const wrapper = shallow(
 );
 
 wrapper.find('.foo').forEach(function (node) {
-  expect(s.hasClass('foo')).to.be true;
+  expect(node.hasClass('foo')).to.be.true;
```

Comment:
would you mind changing this to `.to.equal(true)`? Our styleguide prohibits noop expressions which includes `.to.be.true`